lib/mappers/pa_mapper.py

```diff
@@ -0,0 +1,74 @@
+from dplaingestion.selector import exists, getprop
+from dplaingestion.mappers.dublin_core_mapper import DublinCoreMapper
+from dplaingestion.utilities import iterify
+from dplaingestion.selector import exists, setprop
+
+class PAMapper(DublinCoreMapper):
+
+    dcmi_types = {
+        "Collection",
+        "Dataset",
+        "Event",
+        "Image",
+        "InteractiveResource",
+        "MovingImage",
+        "PhysicalObject",
+        "Service",
+        "Software",
+        "Sound",
+        "StillImage",
+        "Text"
+    }
+
+    def __init__(self, provider_data):
+        super(PAMapper, self).__init__(provider_data)
+
+    def map_provider(self):
+        self.mapped_data.update({"provider": "Pennsylvania Digital Collections Project"})
+
+    def map_relation(self):
+        prop = "relation"
+        if exists(self.provider_data, prop):
+            relations = iterify(self.provider_data.get(prop))
+            self.update_source_resource({"isPartOf": relations[0]})
+            if len(relations) > 1:
+                self.update_source_resource({"relation": relations[1:]})
+
+    def map_type(self):
```

Comment:
@mdellabitta Purely as an FYI but there is an alternative way to do this with the `enrich-type` enrichment. https://github.com/dpla/ingestion/blob/e0778e6afdcbf2eba35a2b3fefbf52dda6fdea97/lib/akamod/enrich-type.py